website/static/js/project-organizer.js

```diff
@@ -364,10 +375,14 @@ var tbOptions = {
                     }
                     tb.filter(event);
                 },
-                onchange: m.withAttr('value', tb.filterText),
+                onchange: function() {
+                    m.withAttr('value', tb.filterText);
+                    $osf.trackClick('myProjects', 'filter', 'search-projects');
```

Comment:
We probably don't want to track *every* change--the user may be typing a lot here, and we'll get a lot of noise. You can use the [debounce](https://github.com/CenterForOpenScience/osf.io/blob/54418bc5fbce98fdb892267a27a93a560b06433a/website/static/js/osfHelpers.js#L347) function to make sure that trackClick only gets called after the user stops typing for some time interval (e.g. 500ms).